src/test/java/com/smartbear/readyapi/client/DataGenDataSourceTest.java

```diff
@@ -76,4 +86,210 @@ public void buildsRecipeWithDataSourceTestStepWithBooleanDataGenWithDigitsFormat
         assertThat(dataGenerator.getFormat(), is(BooleanDataGenerator.FormatEnum._1_0));
     }
 
+    @Test
+    public void buildsRecipeWithDataSourceTestStepWithCityDataGenDataSource() throws Exception {
+        TestRecipe recipe = newTestRecipe()
+                .addStep(dataGenDataSource()
+                        .withNumberOfRows(13)
+                        .withProperty(
```

Comment:
if I want to add multiple properties - could we add a withProperties(...) method that would allow:

```
withProperties( 
    emailTypeProperty( "email" ),
    cityTypeProperty( "city")
)
```
?